### DISCUSSION TITLE
Custom thresholds based on time

### DISCUSSION THREAD

**Discussion by gulo101** [blakeblackshear/frigate#2568] [Ideas]
Hi

Not sure if this was discussed before but would it be possible to define min score / thresholds for each object (per camera) based on time range? For example my driveway camera marks cars in the 80-90% range during the day, but only 50-60% at night, resulting in no alerts. I would like to lower the threshold at night. Thanks!

![image](https://user-images.githubusercontent.com/11531549/147893775-7d751988-ceb8-43a8-84d8-2987cd9e6eea.png)
![image](https://user-images.githubusercontent.com/11531549/147893781-c2345d4b-36c1-43b3-af8e-730b2b76f5c4.png)

**Comment by blakeblackshear** [comment]
You could do this for notifications by using conditions, but this is a reasonable feature request. Please open an actual issue with this request if you don't see one already.

**Reply by gulo101** [reply]
Thanks I will open a request.
